BookEcommerce.Models/DTOs/ProductDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using BookEcommerce.Models.DTOs.Response.Base;
+using BookEcommerce.Models.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookEcommerce.Models.DTOs
+{
+    public class ProductDTO : ResponseBase
+    {
+        public string? ProductId { get; set; }
```

Comment:
those properties belong to Product (Product.Name is better than Product.ProductName). And I'm not a big fan of adding suffix like DTO after object name. Should be using ProductRequestModel/ProductResponseModel/ProductModel....